### ISSUE TITLE
Create sample Profile page

### ISSUE THREAD

**Issue by HumaneIsTaken** [HumaneIsTaken/literate-octo-umbrella#5]
Create a small profile page, and test to make sure it only shows for the specific user.

**Comment by HumaneIsTaken** [comment]
Maybe create a new table 'ExtendedProfile' which has all extra information which may be required.

For now it can be the following:

- UserName -> Self-explanatory, PK/FK (Create Issue to add UserName to main Users table)
- DisplayName -> Name used through-out the site
- DateJoined - > Self-explanatory.
- DisplayPicture -> Picture which user can upload to serve as their display-picture, have a default one somewhere.
- Bio -> Small area of text used to describe the user.

New issue (ISSUE HERE) must be completed before work on this can be done.

**Comment by HumaneIsTaken** [comment]
The previous comment details the possible measures that could be taken to facilitate profiles in the database. However, I must now plan out how the profile page will be displayed and will it be displayed as the landing page after logging in? Making a separate issue for landing page.
